aws/resource_aws_spot_fleet_request.go

```diff
@@ -1019,9 +1215,53 @@ func resourceAwsSpotFleetRequestRead(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("error setting tags: %s", err)
 	}
 
+	if len(config.LaunchTemplateConfigs) > 0 {
+		d.Set("launch_template_configs.0.launch_template_specification.0", flattenFleetLaunchTemplateSpecification(config.LaunchTemplateConfigs[0].LaunchTemplateSpecification))
+		d.Set("launch_template_configs.0.overrides", setLaunchTemplateOverrides(config.LaunchTemplateConfigs[0].Overrides))
+	} else {
+		d.Set("launch_template_configs.0.launch_template_specification.0", nil)
+	}
```

Comment:
When using `d.Set()` with aggregate types (TypeList, TypeSet, TypeMap), [we should perform error checking to prevent issues where the code is not properly able to set the Terraform state](https://www.terraform.io/docs/extend/best-practices/detecting-drift.html#error-checking-aggregate-types). 

These will likely be returning an error currently, or if they aren't, the logic should show as invalid during `ImportStateVerify` when there are multiple `launch_template_config` since nothing beyond the first element is being set.